pixeltable/exec/cache_prefetch_node.py

```diff
@@ -26,16 +29,16 @@ class CachePrefetchNode(ExecNode):
     """Brings files with external URLs into the cache
 
     TODO:
-    - adapting the number of download threads at runtime to maximize throughput
+    - Adapt the number of download threads at runtime to maximize throughput
```

Comment:
Remove this, a fixed number is fine.

Add instead: Create asyncio.Tasks to consume our input in order to increase concurrency.